src/ui/public/doc_table/doc_table.js

```diff
@@ -102,13 +104,38 @@ uiModules.get('kibana')
           if ($scope.searchSource !== $scope.searchSource) return;
 
           $scope.hits = resp.hits.hits;
+          // We limit the number of returned results, but we want to show the actual number of hits, not
+          // just how many we retrieved.
+          $scope.totalHitCount = resp.hits.total;
+          $scope.pager = pagerFactory.create($scope.hits.length, 50, 1);
+          calculateItemsOnPage();  // eslint-disable-line no-use-before-define
 
           return $scope.searchSource.onResults().then(onResults);
         }).catch(notify.fatal);
 
         $scope.searchSource.onError(notify.error).catch(notify.fatal);
       }));
 
+      const limitTo = $filter('limitTo');
+      const calculateItemsOnPage = () => {
```

Comment:
How about moving `limitTo` and `calculateItemsOnPage` up to line 53 where the other non-scope variables are? This way the non-scope and scope variables/methods will be grouped together, and you can remove the `eslint-disable-line` annotation on 111.